src/ulaw.c

```diff
@@ -827,7 +828,9 @@ s2ulaw_array (const short *ptr, int count, unsigned char *buffer)
 static inline void
 i2ulaw_array (const int *ptr, int count, unsigned char *buffer)
 {	while (--count >= 0)
-	{	if (ptr [count] >= 0)
+	{	if (ptr [count] == INT_MIN)
+			buffer [count] = ulaw_encode [INT_MAX >> (16 + 4)] ;
```

Comment:
Why is that `16 + 4` instead of `16 + 2` as used elsewhere in this function?

Reply:
My bad, thanks for catching that. Fixed.